Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN yarn --production --pure-lockfile
 COPY . /app/
 
 #expose default port of the docker to 3000
-EXPOSE 3000
+EXPOSE 3030
```

Comment:
@ScionOfBytes `3000` is quite common port and generally not available. For example aerospike by default runs on 3000. So decided to use any uncommon port.